app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+	def create
+		# byebug
+		@comment = current_user.comments.new(comment_params)
+		if @comment.save
+			flash[:success] = "Comment successfully created"
+			redirect_to root_url
+		else
+			flash[:error] = "Something went wrong"
+			render 'new'
+		end
+	end
+
+	def show
+		comment = Comment.find_by_id(params[:id])
+		if(comment.present?)
```

Comment:
If there are not much arguments then we can omit parenthesis specially in Ruby
so it'd become:
```ruby
if comment.present?
```